app/controllers/v2/link_changes_controller.rb

```diff
@@ -0,0 +1,9 @@
+module V2
+  class LinkChangesController < ApplicationController
+    def search
+      result = LinkChange
```

Comment:
Can we add a filter for link_type here too? That would prevent us from having to deal with organisations/topic/policies tagging elsewhere.